index.js

```diff
@@ -3,6 +3,23 @@ function titleToId(title) {
 	return title.trim().replace(/^\$\.|\(\)$/g, "");
 }
 
+function sourceFromMethodName(fn) {
```

Comment:
I just realized a name that would be more in line with the method above it would be `methodNametoSource`. Let me know if you want that changed.